Fix:

error: implicit declaration of function 'conv_enc' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'conv_init' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- common/conv.h.orig	2023-09-22 20:14:30.000000000 -0700
+++ common/conv.h	2023-09-22 20:15:09.000000000 -0700
@@ -21,3 +21,6 @@
 	char2wchar_t	input2int;
 	wchar2char_t	int2disp;
 };
+
+void conv_init (SCR *orig, SCR *sp);
+int conv_enc (SCR *sp, int option, char *enc);
